routes/web.php

```diff
@@ -42,13 +44,21 @@
     ->name('manager.')
     ->group(function () {
         // Team関係
-        Route::get('/teams', [TeamController::class, 'index'])->name('teams.index');
-        Route::get('/teams/create', [TeamController::class, 'create'])->name('teams.create');
         Route::get('/teams/{team}', [TeamController::class, 'show'])->name('teams.show');
         Route::get('/teams/{team}/edit', [TeamController::class, 'edit'])->name('teams.edit');
-        Route::post('/teams/store', [TeamController::class, 'store'])->name('teams.store');
         Route::patch('/team/{team}', [TeamController::class, 'update'])->name('teams.update');
 
         // Task関係
         Route::resource('/teams.tasks', TaskController::class);
+
+        // Member関係
+        Route::resource('/teams.members', MemberController::class);
+    });
+
+Route::middleware(['auth'])
+    ->prefix('teams')
+    ->group(function () {
+        Route::get('/', [ControllersTeamController::class, 'index'])->name('teams.index');
+        Route::get('/create', [ControllersTeamController::class, 'create'])->name('teams.create');
+        Route::post('/', [ControllersTeamController::class, 'store'])->name('teams.store');
     });
```

Comment:
これ系の場合には prefix なしで書くスタイルの方がいいですね。イメージとしてはこんな感じで

```suggestion
Route::middleware(['auth'])->group(function () {
  Route::resource('/teams', App\Http\Controllers\TeamController::class, ['only' => ['index', 'create', 'store']]);
});
```

リソースの塊を書きたい時には resource を使い、シンプルに階層を作りたい時に prefix を使うと理解しやすいコードになります。

`['only' => ['index', 'create', 'store']` については今日拾う予定だったのですが忘れてしまってました。こんなふうにして指定のパスだけ作ることができます。

Reply:
勉強になります。
ありがとうございます！